lua/mason-registry/glint-language-server/init.lua

```diff
@@ -0,0 +1,16 @@
+local Pkg = require "mason-core.package"
+local npm = require "mason-core.managers.npm"
+
+return Pkg.new {
+    name = "glint-language-server",
+    desc = [[Glint Language server, for Glimmer-flavored JavaScript and TypeScript]],
+    homepage = "https://github.com/typed-ember/glint/tree/main/packages/core",
```

Comment:
```suggestion
    homepage = "https://typed-ember.gitbook.io/glint/",
```

Reply:
Ah, this is supposed to be docs, rather than language-server code location. thanks!